client/shipping-label/state/reducer.js

```diff
@@ -422,21 +422,24 @@ reducers[ SET_PACKAGE_TYPE ] = ( state, { packageId, boxTypeId } ) => {
 	const newPackages = { ...state.form.packages.selected };
 	const oldPackage = newPackages[ packageId ];
 
+	const oldBox = state.form.packages.all[ oldPackage.box_id ];
+	const newBox = state.form.packages.all[ boxTypeId ];
+	const weight =
```

Comment:
also, I'm guessing that the kinds of errors we will get from floating-point oddities will be far lower than what would be significant with the weight of any shippable thing. if there is extreme concern then we can safely store no decimal places, instead using an integer and choosing appropriate units